modules/gtdbtk.nf

```diff
@@ -43,7 +43,19 @@ process GTDBTK {
     --extension fna \
     --skip_ani_screen \
     --out_dir gtdbtk_results
-
+    
+    process_gtdb_unknowns.py -i gtdbtk_results -p processed
+    
+    if [ -e gtdbtk_results/classify/processed_gtdbtk.bac120.summary.tsv ]
```

Comment:
probably it makes sense to rename `gtdbtk_results/classify/gtdbtk.bac120.summary.tsv` to `gtdbtk_results/classify/pre-processed_gtdbtk.bac120.summary.tsv` or similar inside **process_gtdb_unknowns.py**. And then `process_gtdb_unknowns.py` can create files with final names: `gtdbtk_results/classify/gtdbtk.bac120.summary.tsv` and `gtdbtk_results/classify/gtdbtk.ar53.summary.tsv`. 
Movings in nextflow script can cause problems. I guess it is easier to control in python script.

Reply:
I thought about this but I don't think we need them -  we don't share raw results with users (i.e the contents of this folder) and we can't work with differently formatted taxonomies, they break many things. Reformatting does not fundamentally change the meaning here so I think it's ok.

Reply:
OK, but do we keep a log file with which lines we have changed? That could be useful to trace back errors in the future.

Reply:
Added a log for replacements.